src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_executable.py

```diff
@@ -19,28 +18,25 @@
 class TestExecutable:
     def test_flow_build_executable(self):
         source = f"{FLOWS_DIR}/web_classification/flow.dag.yaml"
-        try:
-            with tempfile.TemporaryDirectory() as temp_dir:
-                run_pf_command(
-                    "flow",
-                    "build",
-                    "--source",
-                    source,
-                    "--output",
-                    temp_dir,
-                    "--format",
-                    "executable",
-                )
-                check_path_list = [
-                    "flow/flow.dag.yaml",
-                    "connections/azure_open_ai_connection.yaml",
-                    "pf.bat",
-                    "pf",
-                    "start_pfs.vbs",
-                ]
-                output_path = Path(temp_dir).resolve()
-                for check_path in check_path_list:
-                    check_path = output_path / check_path
-                    assert check_path.exists()
-        finally:
```

Comment:
why removed shutil part?

Reply:
no need, since we use TemporaryDirectory which will remove it automatcally